packages/default/scss/calendar/_layout.scss

```diff
@@ -598,6 +602,11 @@
                 }
             }
 
+            > :first-child,
```

Comment:
The buttons have classes, we should use those!

For now I am letting the PR ahead